src/core/electrostatics_magnetostatics/fft.cpp

```diff
@@ -283,12 +283,12 @@ void pack_block_permute1(double const *const in, double *const out,
   auto const s_in_offset = element * (dim[2] * (dim[1] - size[1]));
   /* offset for mid changing indices of output grid */
   auto const m_out_offset = (element * size[0]) - element;
-  /* linear index of in grid, linear index of out grid */
+  /* linear index of in grid */
```

Comment:
is this docstring missing a word?

Reply:
that's the name of the argument `double const *const in`

Reply:
okay, i think this comment is not readable and should go, but it's not important